spec/vc-11.json

```diff
@@ -0,0 +1,79 @@
+{
+  "$schema": "http://json-schema.org/draft-2020-12/schema#",
+  "type": "object",
+  "definitions": {
+    "credentialStatus": {
+          "type": "object",
+          "properties": {
+            "id": { "type": "string" },
+            "type": { "type": "string" },
+            "statusPurpose": { "type": "string" },
+            "statusListIndex": { "type": "string" },
+            "statusListCredential": { "type": "string" }
+          },
+          "required": ["id", "type", "statusPurpose", "statusListIndex", "statusListCredential"]
+        },
+        "credentialSchema": {
```

Comment:
is the formatting off here?

Reply:
we dont have credentialSchema in js currently

Reply:
no problem - this should lead the impls